usr/share/rear/conf/default.conf

```diff
@@ -163,6 +163,21 @@ KERNEL_CMDLINE=""
 COPY_KERNEL_PARAMETERS=( 'net.ifnames' 'biosdevname' )
 ####
 
+####
+# The command that is called after "rear recover" succeeded to recreate the system
+# when "rear recover" is run in 'auto_recover'/'automatic' or 'unattended' mode.
+# Normally 'reboot' is called to 'reboot' the recreated system:
+RECOVERY_REBOOT_COMMAND="reboot"
+# But in 'unattended' mode an automated 'reboot' call could lead to an endless loop
+# when the ReaR recovery system is booted by default by the firmware (BIOS or UEFI)
```

Comment:
Should not happen with UEFI. We restore the boot order at the end of recovery. So, after reboot, the recovered disk should be booted.

Reply:
Fixed via
https://github.com/rear/rear/pull/3070/commits/e177614ceee010c1186d25eed8074198f9f0efeb